### PR TITLE
Added quicksort C implementation

### LINKED ISSUE #2: Quicksort [C]
https://en.wikipedia.org/wiki/Quicksort

### PR DESCRIPTION
**Pull request by aviaryan**
Fixes #2

### CODE REVIEW

File: quicksort/quicksort.c [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+			quicksort(c, &ar[0]);
+		if (ar_size-c-1 != 0)
+			quicksort(ar_size-c-1, &ar[c+1]);
+	}
+}
+
+int main(){
+	int ar_size = 4, i;
+	int a[4];
+	a[0] = 2; a[1] = 3; a[2] = 0; a[3] = 4;
```
Comment (singhpratyush) [inline quicksort/quicksort.c]: Maybe simply 
```C
int a[4] = {2, 3, 0, 4};
```

**Comment by yashLadha** [comment]
@singhpratyush this code looks good. I am closing my reviews.

**Comment by singhpratyush** [comment]
There are still some corrections to be done. @aviaryan : Please take a look at all the comments.

**Comment by aviaryan** [comment]
@singhpratyush Please review

**Comment by singhpratyush** [comment]
Will merge locally because of conflicts

**Comment by aviaryan** [comment]
@singhpratyush There are no conflicts.

**Comment by singhpratyush** [comment]
All set. Thanks.

**Pull request merged**